serverscripts/docker.py

```diff
@@ -4,15 +4,51 @@
 import logging
 import os
 import serverscripts
-import subprocess
 import sys
 
+from serverscripts.utils import get_output
+from serverscripts.checkouts import (
+    parse_freeze,
+    parse_python_version,
+)
```

Comment:
Ik ben heel sterk voor deze stijl omdat het grepbaar is:

```suggestion
from serverscripts.checkouts import parse_freeze
from serverscripts.checkouts import parse_python_version
```